integration-tests/wallet-estimation-tests.spec.ts

```diff
@@ -0,0 +1,506 @@
+import { DEFAULT_FEE, MANAGER_LAMBDA, TezosToolkit, Protocols } from "@taquito/taquito";
+import { Contract } from "taquito/src/contract/contract";
+import { CONFIGS } from "./config";
+import { originate, originate2, transferImplicit2 } from "./data/lambda";
+import { ligoSample } from "./data/ligo-simple-contract";
+import { managerCode } from "./data/manager_code";
+
+CONFIGS().forEach(({ lib, setup, knownBaker, createAddress, protocol, rpc }) => {
+  const Tezos = lib;
+
+  const carthagenet = (protocol === Protocols.PsCARTHA) ? test : test.skip;
+  const delphinet = (protocol === Protocols.PsDELPH1) ? test : test.skip;
+  const edonet = (protocol === Protocols.PtEdo2Zk) ? test : test.skip;
+
+  describe(`Estimate scenario for contract made with wallet api using: ${rpc}`, () => {
+    let LowAmountTez: TezosToolkit;
+    let contract: Contract;
+    const amt = 2000000 + DEFAULT_FEE.REVEAL;
+
+    beforeAll(async (done) => {
+      try {
+        await setup()
+        LowAmountTez = await createAddress();
+        const pkh = await LowAmountTez.signer.publicKeyHash()
+        const transfer = await Tezos.wallet.transfer({ to: pkh, mutez: true, amount: amt }).send();
+        await transfer.confirmation();
+        const op = await Tezos.contract.originate({
```

Comment:
Can we use the wallet API instead?

Reply:
whoops, I see, thanks 👍🏻 )

Reply:
changed to use the wallet api throughout